features/step_definitions/map_steps.rb

```diff
@@ -104,3 +104,7 @@ def stub_request_with_address(address, body = nil)
   org.latitude.should_not be_nil
   org.longitude.should_not be_nil
 end
+
+Then(/^I should see the map key$/) do
+  expect(page).to have_text("key")
```

Comment:
This isn't really a the assertion we'd want since (1) the user cannot see the key text as its been made to have no display by the CSS, and (2) the tip text about 12 months is what is important.